src/main/resources/interface-specification.yml

```diff
@@ -1574,6 +1574,37 @@ paths:
               schema:
                 type: integer
               description: Time interval in milliseconds. Allowed requests will be constantly replenished during the interval. At the end of the interval the max allowed requests will be available
+  /eservices:
+    parameters:
+      - $ref: '#/components/parameters/CorrelationIdHeader'
+      - $ref: '#/components/parameters/IpAddress'
+    post:
+      security:
+        - bearerAuth: [ ]
+      tags:
+        - catalog
+      summary: Create a new e-service
+      operationId: createEService
+      requestBody:
+        description: An E-Service seed
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/EServiceSeed'
+        required: true
+      responses:
+        '200':
+          description: EService created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/OldEService'
```

Comment:
(confirmed by @Carminepo2 ) we need to return just the id of the new eservice, you can use the `CreatedResource` model

Btw, models named `Old*` are just intended as a temporary measure in the catalog process.
Names in the BFF should not be named with `Old` prefix (and these specific models are not needed here)